eprihlaska/ais_utils.py

```diff
@@ -399,6 +414,8 @@ def save_application_form(ctx,
     with app.collect_operations() as ops:
         app.d.enterButton.click()
 
+    print("Post submit: {}".format(ops), file=sys.stderr)
```

Comment:
Do we need these `print` statements, or they are here just for easier debugging failed requests?

Reply:
These are indeed just for debugging

Reply:
Is it such a big problem to leave them in? (After all, more diagnostic info leads to easier debugging...)

Reply:
@tvinar what Adman was probably alluding to is the fact that this is strictly "the wrong" way of doing it. That said, I do think we can live with this for sure.